site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -2215,6 +2218,8 @@ public function ajaxGetOverallComment($gradeable_id, $anon_id = '') {
         if ($submitter_id === false) {
             return;
         }
+        // This function always retrieves the overall comment for the currently logged in grader.
```

Comment:
Get rid of this comment. It's more or less the same as the one above the function.

Reply:
Removed